suite-native/atoms/src/DiscreetText.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { LayoutChangeEvent } from 'react-native';
+
+import { useAtom, atom } from 'jotai';
+import { Blur, Canvas, Text as SkiaText, useFont } from '@shopify/react-native-skia';
+
+import { Color, TypographyStyle, typographyStylesBase } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Text } from './Text';
+import { Box } from './Box';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Regular.otf');
+
+type DiscreetTextProps = {
+    typography?: TypographyStyle;
+    color?: Color;
+    text: string;
```

Comment:
Can you make this `children`? I think it makes more sense, because it should have same API as Text component.